myWords.txt

```diff
@@ -0,0 +1,14 @@
+customizability
+Gantt
+jison
+knsv
+Knut
+mindmap
+Mindmaps
+mitigations
+sandboxed
+Sveidqvist
+verdana
+Visio
+vitepress
+colour
```

Comment:
By the way, you don't need this file.

Just add any words to `cSpell.json`'s `words` array, and cspell eslint should automatically load it:

https://github.com/mermaid-js/mermaid/blob/9ac3992fd2e9b679d79b9e798c0918253d42f608/cSpell.json#L4-L17

Reply:
it was not loading automatically. so I added this file.

Reply:
It seems to works fine on my computer if I delete the `myWords.txt` file and instead add works to `cSpell.json`.

It might be worth deleting the `myWords.txt` file and checking again if it works. If it still doesn't load the words automatically from `cSpell.json`, never mind; maybe it's just a Linux/MacOS/Windows difference.

Reply:
You are right. I was using `--cache`  so it was not working properly. I tested it now and added some words to ignore words list. I have also pushed many eslint error fixes. There are still a few and I couldn't judge if I have to add those words to ignore words or just use eslint-disable since it occurred only once. Take a look once now.

Reply:
Awesome! Thanks for all your work, I went ahead and fixed the rest of the cspell eslint warnings, see my main comment: https://github.com/mermaid-js/mermaid/pull/3604#pullrequestreview-1143273787